var/spack/repos/builtin/packages/glm/package.py

```diff
@@ -13,26 +13,34 @@ class Glm(CMakePackage):
 
     homepage = "https://github.com/g-truc/glm"
     url = "https://github.com/g-truc/glm/archive/0.9.9.8.tar.gz"
+    git = "https://github.com/g-truc/glm.git"
 
+    version('develop', branch="master")
     version('0.9.9.8', sha256='7d508ab72cb5d43227a3711420f06ff99b0a0cb63ee2f93631b162bfe1fe9592')
     version('0.9.7.1', sha256='285a0dc8f762b4e523c8710fbd97accaace0c61f45bc8be2bdb0deed07b0e6f3')
 
     depends_on('cmake@2.6:', type='build')
     depends_on('cmake@3.2:', type='build', when='@0.9.9.0:')
 
     # CMake install target was removed in version 0.9.9.6
-    @when('@0.9.9.6:')
+    @when('@0.9.9.6:0.9.9.8')
```

Comment:
Ok, I am confused. Has CMake only been removed from 0.9.9.6 to 0.9.9.8 and it is restored in the develop version of the package?

Reply:
Precisely, [removed](https://github.com/g-truc/glm/commit/5f352ecce21bb1ab37fa56fac0f383c779b351a3) and [added](https://github.com/g-truc/glm/pull/1054).

Reply:
Thanks for clarifying.